index.html

```diff
@@ -34,6 +34,112 @@ <h3>LET'S CONNECT</h3>
    <li><a title="Angelist" href="https://angelist.com/Ashna_Ali1"><img src="./Angel.jpg" alt="angelist"></a></li>
   </ul>
  </section>
+ <section class="projects">
+<div class="project">
+  <img class="picture" src="First image.jpg" alt="project1">
+  <h2 class="post">Tonic</h2>
+  <ul class="toni-flex">
+    <li class="link">Canopy</li>
+    <li><img src="./Counter.jpg" alt="counter">Back End Dev</li>
+    <li><img src="./Counter.jpg" alt="counter">2023</li>
+  </ul>
+<p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+<ul class="list-flex">
+<li class="language">html</li>
+<li class="language">css</li>
+<li class="language">javascript</li>
+</ul>
+<button><strong>See Project</strong></button>
+</div>
+ </section>
+
+ <section class="projects">
+  <div class="project">
+    <img class="picture" src="Second image.jpg" alt="project1">
+    <h2 class="post">Tonic</h2>
+    <ul class="toni-flex">
+      <li class="link">Canopy</li>
+      <li><img src="./Counter.jpg" alt="counter">Back End Dev</li>
+      <li><img src="./Counter.jpg" alt="counter">2023</li>
+    </ul>
+  <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+  <ul class="list-flex">
+  <li class="language">html</li>
+  <li class="language">css</li>
+  <li class="language">javascript</li>
+  </ul>
+  <button><strong>See Project</strong></button>
+  </div>
+   </section>
+
+   <section class="projects">
+    <div class="project">
+      <img class="picture" src="Third image.jpg" alt="project1">
+      <h2 class="post">Tonic</h2>
+      <ul class="toni-flex">
+        <li class="link">Canopy</li>
+        <li><img src="./Counter.jpg" alt="counter">Back End Dev</li>
+        <li><img src="./Counter.jpg" alt="counter">2023</li>
+      </ul>
+    <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="list-flex">
+    <li class="language">html</li>
+    <li class="language">css</li>
+    <li class="language">javascript</li>
+    </ul>
+    <button><strong>See Project</strong></button>
+    </div>
```

Comment:
- [ Required ] Great job :clap: working on the `projects` section it looks good, just one observation that i think will improve your project. Please consider reducing the `margin-top` and `margin-top` properties between successive project cards, currently, it's bigger than what is provided on the `Figma template`

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/212340492-7c603ffe-ad89-479a-8a16-164cd88a8a6a.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/212340668-4f32e811-3162-488a-af81-8e1332b5ae89.png" width="375px" />